src/renderer/pages/Apps.tsx

```diff
@@ -132,7 +132,7 @@ export default function Apps() {
   const nodeConfigRelease = useSelector(selectConfig).release;
   const dispatch = useAppDispatch();
 
-  const [showTooltip, setShowTooltip] = useState(false);
+  const [, setShowTooltip] = useState(false);
```

Comment:
if we are not using the state anywhere, why do we need to set it?

Reply:
i think it is used for this part

const handleWarningIconHover = () => {
    setShowTooltip(true);
  };

  const handleWarningIconLeave = () => {
    setShowTooltip(false);
  };